package.json

```diff
@@ -158,13 +158,26 @@
     "@aws-sdk/client-s3": "^3.412.0",
     "@babel/core": "^7.14.6",
     "@babel/register": "^7.14.5",
+    "@codemirror/autocomplete": "^6.18.0",
+    "@codemirror/commands": "^6.6.1",
+    "@codemirror/lang-css": "^6.3.0",
+    "@codemirror/lang-javascript": "^6.2.2",
+    "@codemirror/language": "^6.10.3",
+    "@codemirror/lint": "^6.8.1",
+    "@codemirror/search": "^6.5.6",
+    "@codemirror/state": "^6.4.1",
+    "@codemirror/view": "^6.33.0",
     "@emmetio/codemirror-plugin": "^1.2.4",
+    "@emmetio/codemirror6-plugin": "^0.3.2",
     "@gatsbyjs/webpack-hot-middleware": "^2.25.3",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.11",
     "@redux-devtools/core": "^3.11.0",
     "@redux-devtools/dock-monitor": "^3.0.1",
     "@redux-devtools/log-monitor": "^4.0.2",
     "@reduxjs/toolkit": "^1.9.3",
+    "@replit/codemirror-css-color-picker": "^6.2.0",
+    "@simonwep/pickr": "^1.9.1",
+    "@uiw/react-codemirror": "^4.23.1",
```

Comment:
if we're not using this one let's remove it!

Reply:
removed most but left this @replit/codemirror-css-color-picker to see if it is useful to implement the color picker!